code-of-conduct.md

```diff
@@ -0,0 +1,84 @@
+
+It may be appropriate for a recruitment company to sponsor refreshments and do a quick pitch to a relevant audience of potential candidates, but this is down to the discretion of the event organiser(s). Recruiting during an event without prior arrangement will be treated as a breach of this code of conduct.
+
+### What to do if you have a concern
+
+If you are being harassed or feel uncomfortable, notice that someone else is being harassed, or have any other concerns, please speak to the event organiser(s) immediately. You can also contact the event organiser through Meetup and Slack.
+
+If you are not able to to do this or you feel that your concern is not being taken seriously then please contact the Equity Officer. This is currently Simon Davey, who can be reached through Slack or by email (simonr.davey@gmail.com).
+
+CodeWith is here to listen to your concerns or questions and you don’t have to make a formal report unless you want to. You do not have to give us details of the harassment.
+
+If you wish to report it, we will take details of the harassment and work with you to respond to the issue in a way that helps you feel safe.
+
+If necessary, the event organisers will help participants contact police or venue security, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the event.
+
+### Where and when this code of conduct applies
+
+We expect participants to follow this code of conduct at all our events, partner events, related social events, such as post-event drinks, networking discussions, meetups and workshops. Event sponsors must also follow this anti-harassment policy, which includes not using sexualised images, activities, or other material.
+
+### Exceptions
+
+Any discussion or images that relate to sex, pornography, discriminatory language, or similar must meet the following criteria.
+
+- The organisers must specifically grant permission in writing.
+- It must be necessary to the topic of discussion and no alternative exists.
+- It must be presented in a respectful manner.
+- Attendees must be warned in advance and respectfully given plenty of warning and the opportunity to leave beforehand.
+- This exception specifically does not allow the use of gratuitous sexual images as a device to grab attention or for unnecessary examples.
+
+Thank you
+
+This document was based on the [Sheffield Digital Code of Conduct](https://sheffield.digital/events/meetup-code-of-conduct/), which in turn is based on the work of JSConf, Geek Feminism, confcodeofconduct.com, and inspired by resources provided by Ashe Dryden.
+
+Licensed under a Creative Commons Attribution 3.0 Unported License.
+Version 1
+
+This is version 1 of the CodeWith Code of Conduct. It was last updated by Simon Davey on 31 August 2020. If you think we have made any errors or omissions, please contact us by email (contact@codewith.org.uk) or via the [contact form](contact.html).
```

Comment:
Our contact form doesn't function in a way that lets people send us a message. Perhaps word it as being possible to contact us through one of the methods *on* the contact form?

Reply:
Addressed.